.travis.yml

```diff
@@ -109,10 +117,16 @@ env:
     - MINCONDA_OSX="MacOSX-x86_64"
 
 install:
+  - export HOMEBREW_NO_AUTO_UPDATE=1
```

Comment:
why do you have this complicated logic with first setting a veriable, then checking the parameter and unsetting it and then having the logic below.. i think this can be simplified?

Reply:
This is setting a safer default -> dont automatically update versions